mesh/src/main/java/no/nordicsemi/android/mesh/transport/GenericBatteryStatus.java

```diff
@@ -40,9 +40,9 @@ void parseStatusParameters() {
         final ByteBuffer buffer = ByteBuffer.wrap(mParameters).order(ByteOrder.LITTLE_ENDIAN);
```

Comment:
We don't need a byte buffer here, as the endianness will not matter as we are just dealing with single octets.

Reply:
You are right. I will remove this line and work with mParameters directly.